src/include/sof/low_latency_schedule.h

```diff
@@ -81,23 +81,23 @@ static inline void work_init(struct work *w,
 		w->flags = xflags;
 }
 
-struct work_queue **arch_work_queue_get(void);
+struct ll_queue **arch_ll_queue_get(void);
 
 /* schedule/cancel work on work queue */
-void work_schedule(struct work_queue *queue, struct work *w, uint64_t timeout);
-void work_reschedule(struct work_queue *queue, struct work *w, uint64_t timeout);
-void work_cancel(struct work_queue *queue, struct work *work);
+void ll_schedule(struct ll_queue *queue, struct ll_task *w, uint64_t timeout);
+void ll_reschedule(struct ll_queue *queue, struct ll_task *w, uint64_t timeout);
+void ll_cancel(struct ll_queue *queue, struct ll_task *work);
 
 /* schedule/cancel work on default system work queue */
-void work_schedule_default(struct work *work, uint64_t timeout);
-void work_reschedule_default(struct work *work, uint64_t timeout);
-void work_reschedule_default_at(struct work *w, uint64_t time);
-void work_cancel_default(struct work *work);
+void ll_schedule_default(struct ll_task *work, uint64_t timeout);
```

Comment:
We should probably change our "timeout" variable name to also reflect the scheduler API timebase used.

Reply:
On PR's HEAD it is done

Reply:
done